pkgs/by-name/op/opensc/package.nix

```diff
@@ -35,43 +34,36 @@ stdenv.mkDerivation rec {
     autoreconfHook
     libxslt # xsltproc
   ];
-  buildInputs =
-    [
-      zlib
-      readline
-      openssl
-      libassuan
-      libXt
-      libiconv
-      docbook_xml_dtd_412
-    ]
-    ++ lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.Carbon
-    ++ (if withApplePCSC then [ darwin.apple_sdk.frameworks.PCSC ] else [ pcsclite ]);
+  buildInputs = [
+    zlib
+    readline
+    openssl
+    libassuan
+    libXt
+    libiconv
+    docbook_xml_dtd_412
+  ] ++ lib.optional (!withApplePCSC) pcsclite;
 
   env.NIX_CFLAGS_COMPILE = "-Wno-error";
 
-  configureFlags = [
-    "--enable-zlib"
-    "--enable-readline"
-    "--enable-openssl"
-    "--enable-pcsc"
-    "--enable-sm"
-    "--enable-man"
-    "--enable-doc"
-    "--localstatedir=/var"
-    "--sysconfdir=/etc"
-    "--with-xsl-stylesheetsdir=${docbook_xsl}/xml/xsl/docbook"
-    "--with-pcsc-provider=${
-      if withApplePCSC then
-        "${darwin.apple_sdk.frameworks.PCSC}/Library/Frameworks/PCSC.framework/PCSC"
-      else
-        "${lib.getLib pcsclite}/lib/libpcsclite${stdenv.hostPlatform.extensions.sharedLibrary}"
-    }"
-  ];
+  configureFlags =
+    [
+      "--enable-zlib"
+      "--enable-readline"
+      "--enable-openssl"
+      "--enable-pcsc"
+      "--enable-sm"
+      "--enable-man"
+      "--enable-doc"
+      "--localstatedir=/var"
+      "--sysconfdir=/etc"
+      "--with-xsl-stylesheetsdir=${docbook_xsl}/xml/xsl/docbook"
+    ]
+    ++ lib.optional (!withApplePCSC)
+      "--with-pcsc-provider=${lib.getLib pcsclite}/lib/libpcsclite${stdenv.hostPlatform.extensions.sharedLibrary}";
 
   PCSC_CFLAGS = lib.concatStringsSep " " (
     lib.optionals withApplePCSC [
-      "-I${darwin.apple_sdk.frameworks.PCSC}/Library/Frameworks/PCSC.framework/Headers"
       "-I${lib.getDev pcsclite}/include/PCSC"
```

Comment:
Uh, do we really want to be using `pcsclite` headers when using Apple’s PCSC rather than `pcsclite`? I find this highly dubious; does it fail without?